3rd-party/class.jetpack-amp-support.php

```diff
@@ -31,6 +31,7 @@ public static function init() {
 		// Sharing.
 		add_filter( 'jetpack_sharing_display_markup', array( 'Jetpack_AMP_Support', 'render_sharing_html' ), 10, 2 );
 		add_filter( 'sharing_enqueue_scripts', array( 'Jetpack_AMP_Support', 'amp_disable_sharedaddy_css' ) );
+		add_action( 'wp_head', array( 'Jetpack_AMP_Support', 'amp_enqueue_sharing_css' ) );
```

Comment:
Could we enqueue this into `wp_enqueue_scripts` instead?

Reply:
Yep sure, will update shortly!